### PR TITLE
feat(web): add iOS and Android home screen support

### PR DESCRIPTION
**Pull request by hiroppy**
## Summary

- add an installable Web App Manifest with deployment-relative launch paths and maskable Android icons
- add iOS standalone metadata and a 180px Apple touch icon
- generate a safe-zone-aware pixel-art app icon from the existing project logo
- verify manifest properties and shipped PNG dimensions with unit tests

## Validation

- `pnpm --filter @mf-dashboard/web test:unit -- src/app/manifest.test.ts` (368 tests)
- `pnpm turbo typecheck` (9 workspaces)
- `pnpm lint`
- `pnpm format:check`
- `pnpm --filter @mf-dashboard/web test:storybook` (330 tests)
- demo runtime with iOS and Android user agents: dashboard, manifest, Apple metadata, and all icon assets returned successfully

## Manual QA Plan

Quality characteristics selected from ISO/IEC 25010:2023:

- Functional suitability: confirm the manifest contains standalone launch settings and required icon sizes.
- Compatibility: inspect iOS Apple metadata and Android manifest behavior separately.
- Usability: confirm the mobile dashboard remains usable when launched through the standalone start path.

Test techniques:

- Equivalence partitioning: validate iOS metadata and Android manifest/icon paths as separate platform classes.
- Boundary value analysis: validate exact 180px, 192px, and 512px icon dimensions.
- Exploratory testing: load the demo dashboard with iPhone and Android user agents and inspect the delivered head/manifest assets.

Out of scope: offline behavior and push notifications; this ticket only adds Home Screen launch metadata and icons.

## Media

iPhone 15 demo dashboard

Linear: [HIR-135](https://linear.app/hiroppy/issue/HIR-135/ios-android%E3%81%A7%E3%83%9B%E3%83%BC%E3%83%A0%E7%94%BB%E9%9D%A2%E3%81%8B%E3%82%89%E3%82%A2%E3%82%AF%E3%82%BB%E3%82%B9%E3%81%A7%E3%81%8D%E3%82%8B%E3%82%88%E3%81%86%E3%81%AB%E3%81%99%E3%82%8B)

### CODE REVIEW

File: apps/web/src/app/layout.tsx [hunk 1]
```diff
@@ -15,13 +15,24 @@ const metadataBase =
...
     default: "MoneyForward Me Dashboard",
   },
   description: "MoneyForward Me のデータを可視化するダッシュボード",
+  manifest: `${basePath}/manifest.webmanifest`,
+  icons: {
+    apple: `${basePath}/apple-touch-icon.png`,
+  },
+  appleWebApp: {
+    capable: true,
+    statusBarStyle: "black-translucent",
```
Comment (hiroppy) [inline apps/web/src/app/layout.tsx]: `black-translucent` makes iOS standalone apps draw content under the status bar, but this PR does not add `viewport-fit=cover` or any `env(safe-area-inset-top)` offset for the fixed header. On devices with a notch/status bar, the installed app header can be obscured. Either use a non-translucent status bar style (`black`/`default`) or add safe-area-aware viewport/header spacing with a regression check.

Comment (hiroppy) [reply apps/web/src/app/layout.tsx]: Addressed in 5ca10b9 by switching the iOS standalone status bar style to non-translucent `black`. This keeps the status bar from overlaying the fixed header without introducing safe-area layout changes. Unit tests, all-workspace typecheck, lint, and format checks pass after the change.

[the remark this answers is not in the document]
Comment (hiroppy) [reply apps/web/src/app/layout.tsx]: Addressed in 5ca10b9 by using the non-translucent black status bar style, preventing the fixed header from being overlaid without adding safe-area layout changes.

File: apps/web/src/app/layout.tsx [hunk 2]
```diff
@@ -15,13 +15,24 @@ const metadataBase =
...
 export const metadata: Metadata = {
   metadataBase,
   title: {
     template: "%s | MoneyForward Me Dashboard",
     default: "MoneyForward Me Dashboard",
   },
   description: "MoneyForward Me のデータを可視化するダッシュボード",
+  manifest: `${basePath}/manifest.webmanifest`,
+  icons: {
+    apple: `${basePath}/apple-touch-icon.png`,
```
[the remark this answers is not in the document]
Comment (hiroppy) [reply apps/web/src/app/layout.tsx]: Addressed in 0da723c. Mobile metadata creation is now a pure helper with unit coverage for both the empty local base path and the /mf-dashboard GitHub Pages base path. The tests assert both manifest and Apple touch icon URLs, along with the standalone iOS metadata.

**Comment by hiroppy** [comment]
Commit: https://github.com/hiroppy/mf-dashboard/commit/0da723ce5ae2240a1b926a98501e92d51a5f15fa

### PWA manifest と iOS メタデータが配信される

manifest.webmanifest が 200 で返り、standalone/start_url/icons と apple-mobile-web-app-* メタデータ、Apple/Android アイコンが取得できることを確認しました。

### デモダッシュボードがホーム画面起動相当のモバイル幅で表示できる

390px 幅でヘッダーとダッシュボードが表示され、横スクロールなしで主要情報を確認できることを確認しました。

pwa-desktop-dashboard
pwa-mobile-dashboard

**Review by hiroppy — COMMENTED** [review]
テストなくていい

**Comment by hiroppy** [comment]
「テストなくていい」を反映し、追加テストのためだけに導入していた補助関数と unit test を削除しました。metadata の値と配信挙動は維持しています。反映コミット: 4d85e82

File: apps/web/src/app/layout.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (hiroppy) [reply apps/web/src/app/layout.tsx:29]: Not restoring this test. The repository owner explicitly reviewed this exact suggestion and instructed 「テストなくていい」, so commit 4d85e82 intentionally removes the test-only helper and coverage while preserving the deployed metadata behavior. The existing manifest and icon tests remain green.
